source/drivers/cxx.txt

```diff
@@ -65,6 +65,11 @@ Compatibility
 MongoDB Compatibility
 ~~~~~~~~~~~~~~~~~~~~~
 
+.. admonition:: Development
+   :class: important
+
+   `*` denotes a version in development.
```

Comment:
Quick question:
Is this version available as a snapshot? I was planning to investigate that as part of the ticket prior to reassignment.

Reply:
I don't have an answer for you at the moment. I'll look into it.